aspnet/conceptual-overview/aspnet.rst

```diff
@@ -85,27 +97,48 @@ You can use any `OWIN <http://owin.org>`_-based middleware with ASP.NET Core. Se
 
 Web root
 --------
 
-The Web root of your application is the root location in your project from which HTTP requests are handled (ex. handling of static file requests). The Web root of an ASP.NET Core application is configured using the "webroot" property in your project.json file.
+The web root of your app is the root location in your project from which HTTP requests are handled (for example handling of static file requests). The web root of an ASP.NET Core app is configured using the ``webroot`` property in the *project.json* file.
+
+Content root
+------------
+
 
 Configuration
 -------------
 
-ASP.NET Core uses a new configuration model for handling of simple name-value pairs that is not based on System.Configuration or web.config. This new configuration model pulls from an ordered set of configuration providers. The built-in configuration providers support a variety of file formats (XML, JSON, INI) and also environment variables to enable environment-based configuration. You can also write your own custom configuration providers. Environments, like Development and Production, are a first-class notion in ASP.NET Core and can also be set up using environment variables:
+ASP.NET Core uses a new configuration model for handling simple name-value pairs. The new configuration model is not based on ``System.Configuration`` or *web.config*; rather, it pulls from an ordered set of configuration providers. The built-in configuration providers support a variety of file formats (XML, JSON, INI) and environment variables to enable environment-based configuration. You can also write your own custom configuration providers.
+
+See :doc:`/fundamentals/configuration` for more information.
+
+Environments
+---------------------
+
+Environments, like "Development" and "Production", are a first-class notion in ASP.NET Core and can  be set using environment variables. See :doc:`/fundamentals/environments` for more information.
 
-.. literalinclude:: /../common/samples/WebApplication1/src/WebApplication1/Startup.cs
-  :language: c#
-  :lines: 22-34
-  :dedent: 12
+Build web UI and web APIs using MVC
+-----------------------------------
 
-See :doc:`/fundamentals/configuration` for more details on the new configuration system and :doc:`/fundamentals/environments` for details on how to work with environments in ASP.NET Core.
+- ASP.NET Core helps you create well-factored and testable web apps that follow the Model-View-Controller (MVC) pattern. See :doc:`/mvc/index` and :doc:`/testing/index`.
```

Comment:
We need another bullet in this list that says you can build HTTP services with ASP.NET Core MVC that support multiple formats and have full support for content negotiation.